ios/FluentUI/Calendar/Views/CalendarViewDayMonthYearCell.swift

```diff
@@ -63,12 +63,13 @@ class CalendarViewDayMonthYearCell: CalendarViewDayMonthCell {
 
     // Only supports indicator levels from 0...4
     func setup(textStyle: CalendarViewDayCellTextStyle,
+               backgroundStyle: CalendarViewDayCellBackgroundStyle,
                selectionStyle: CalendarViewDayCellSelectionStyle,
                monthLabelText: String,
                dateLabelText: String,
                yearLabelText: String,
                indicatorLevel: Int) {
-        super.setup(textStyle: textStyle, selectionStyle: selectionStyle, monthLabelText: monthLabelText, dateLabelText: dateLabelText, indicatorLevel: indicatorLevel)
+        super.setup(textStyle: textStyle, backgroundStyle: backgroundStyle, selectionStyle: selectionStyle, monthLabelText: monthLabelText, dateLabelText: dateLabelText, indicatorLevel: indicatorLevel)
```

Comment:
Nit: break this down into individual paragraphs for readability? (and across code)